### PR TITLE
fixing broken link to Visual E2E WebDriverIO example tests

### PR DESCRIPTION
**Pull request by mattdsauce**
### Description
fixing broken link to Visual E2E WebDriverIO example tests

### Motivation and Context
fixes a broken link (404)

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by lauramthomas — APPROVED** [review]

File: docs/visual/e2e-testing/code-examples.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ import TabItem from '@theme/TabItem';
 
 To view Visual E2E code examples, please refer to the [Sauce Labs Training resources GitHub repo](https://github.com/saucelabs-training):
 
-* [JavaScript (WebDriverIO)](https://github.com/saucelabs-training/demo-js/tree/main/visual-e2e/examples)
+* [JavaScript (WebDriverIO)](https://github.com/saucelabs-training/demo-js/tree/main/webdriverio/webdriver/examples/visual-e2e)
 * [Java](https://github.com/saucelabs-training/demo-java/blob/main/selenium-junit4-examples/src/test/java/com/saucedemo/selenium/junit4/SimpleVisualE2ETest.java)
 * [Python](https://github.com/saucelabs-training/demo-python/tree/main/examples/sauce_visual)
 * [Ruby](https://github.com/saucelabs-training/demo-ruby/tree/main/sauce-features/visual/basic)
```
Comment (kimsauce) [inline docs/visual/e2e-testing/code-examples.md:15]:
```suggestion
* [Ruby](https://github.com/saucelabs-training/demo-ruby/tree/main/sauce-features/visual)
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Minor change

File: docs/visual/e2e-testing/code-examples.md (hunk 1 above)
Comment (kimsauce) [reply docs/visual/e2e-testing/code-examples.md:15]: Hi @mattdsauce , thanks again for submitting this PR! If this additional change looks ok to you, I'll go ahead and merge.

**Comment by mattdsauce** [comment]
looks good, thanks!

**Pull request merged**